File: python/twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i, num in enumerate(nums):
            j = i + 1
            while j < len(nums):
                if num + nums[j] == target:
                    return [i, j]
                else:
                    j+=1

File: go/twoSum.go
func twoSum(nums []int, target int) []int {
    numbers := make(map[int]int)
    for idx, num := range(nums) {
        if digit, found := numbers[target-num]; found {
            return []int{digit, idx}
        }
        numbers[num] = idx 
    }
    return []int{}
}
